tox.ini

```diff
@@ -0,0 +1,68 @@
+[tox]
+envlist = py{35,36,37}-{pytest,setup}{,-all},docs,dev,flake8
+skipsdist = True
+requires = tox-venv
+           setuptools >= 30.0.0
+
+[testenv]
+usedevelop = True
+whitelist_externals = which
+passenv = HOME LC_ALL LC_CTYPE
+description =
+    setup: run tests with python setup.py
+    pytest: run tests with pytest
```

Comment:
Add a description for 'all' that says with all optional dependencies?

Reply:
Done.